assets/src/application/entities/ticket/isOnSale.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { differenceInMinutes } from 'date-fns';
+
+interface IisOnSale {
+	startDate: Date;
+	endDate: Date;
+}
+
+const isOnSale = ({ startDate, endDate }: IisOnSale): boolean => {
+	const now = new Date();
```

Comment:
this could be moved outside of this function since this could potentially be called on every ticket in a list, but we don't need to recalculate "now" for every iteration

Reply:
moved out: https://github.com/eventespresso/event-espresso-core/pull/2065/commits/93c6687ba8294909fa36694945cdf4f3ee8841d2